src/components/SearchPane/SearchPane.tsx

```diff
@@ -9,10 +9,37 @@ import SearchContext from 'SearchPane/Context/SearchPaneContext';
 import SearchPaneForm from './Form/SearchPaneForm';
 import SearchPaneControls from './Controls/SearchPaneControls';
 
+const CSS_CLASSES = {
+  IS_SEARCH_ACTIVE: 'search-open'
```

Comment:
A bit pernickety from me - but what do you think about renaming the selector to something stateful - e.g. `has-search-open`, `has-open-search-pane`, `has-open-search`?

Reply:
I'll change to `is-search-active` (we're using `...active` for other selectors)

Reply:
Done